multi-node/aws/pkg/config/config.go

```diff
@@ -164,6 +166,22 @@ func (c Cluster) Config() (*Config, error) {
 	config.APIServerEndpoint = fmt.Sprintf("https://%s", c.ExternalDNSName)
 	config.K8sNetworkPlugin = "cni"
 
+	// Check if we are running CoreOS 1122.0.0 or greater when using rkt as
+	// runtime. Proceed regardless if running alpha. TODO(pb) delete when rkt
+	// works well with stable.
+	if config.ContainerRuntime == "rkt" && config.ReleaseChannel != "alpha" {
```

Comment:
Can we just remove the second clause of the condition? The check is super cheap already.